File: lista_3/fechadura.py
pinos,meta = map(int, input().split())
alturas = list(map(int, input().split()))

movi = 0

while alturas != []:
      
      dif = meta - alturas[0]
      
      alturas[0] = alturas[0] + dif
      alturas[1] = alturas[1] + dif

      if alturas[1] == meta:
            alturas.pop(1)
      alturas.pop(0)

      if dif < 0:
            dif *= -1
      movi += dif
print(movi)

File: lista_3/fechadura.rb
pinos, meta = gets.split.map(&:to_i)
alturas = gets.split.map(&:to_i)

movimento = 0

while (alturas != [])
      
      	diferenca = meta - alturas[0]
      
      	alturas[0] += diferenca
      	alturas[1] += diferenca

      	if (alturas[1] == meta)
       		alturas.delete_at(1)
      	end 
      
		alturas.delete_at(0)
		movimento += diferenca.abs
		
end    
      
puts movimento
